middleware/crud-functions/crud-function-tests/getTopWinnersTest.js

```diff
@@ -0,0 +1,35 @@
+const getTopWinners = require("../getTopWinners.js");
+const assert = require("chai").assert;
+const should = require("chai").should();
+
+describe("getTopWinners", function() {
+  it("should return a user object with valid attributes", async function() {
+    users = await getTopWinners(1);
```

Comment:
I think we should redeclare users with `let users` it's safer to scope it to each test.